src/Flow/ETL/Stream/Handler.php

```diff
@@ -33,7 +33,7 @@ public function open(FileStream $stream, Mode $mode)
     {
         /** @psalm-suppress PossiblyNullOperand */
         $fullPath = ($this->safeMode)
-            ? (\rtrim($stream->uri(), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . \uniqid() . '.' . $this->extension)
+            ? (\rtrim($stream->uri(), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . \uniqid('', true) . '.' . $this->extension)
```

Comment:
```suggestion
            ? (\rtrim($stream->uri(), DIRECTORY_SEPARATOR) . DIRECTORY_SEPARATOR . \uniqid('', true) . $this->extension)
```

Reply:
removing this dot, would make the code to look like this: `62b89bc101a5f7.27307237csv` instead of `62b89bc101a5f727307237.csv`. It should be something more like: `str_replace(".", "", uniqid("", true))` but lets wait for @stloyd to share his thoughts on this

Reply:
btw thanks for using `Suggest change`, I wasn't aware such a cool thing exists 🙈

Reply:
Replaced with: https://www.php.net/manual/en/function.random-bytes.php

Later we can move all of them to that function.

Reply:
[random_bytes](https://3v4l.org/B2qIG#v8.1.7) would require some hashing, like `\hash('sha256', \random_bytes(23));`

Reply:
@norberttech Done that too fast while watching some serial 😂 Now it's fixed and a little bit improved to reduce amount of checks and psalm warning 😅

Reply:
I was surprised you still had any strength yesterday to write anything lol, thanks! 💪